generators/app/templates/src/FOOBAR/angular.json

```diff
@@ -28,7 +28,9 @@
               "ClientApp/assets",
               "ClientApp/manifest.webmanifest"
             ],
-            "styles": ["ClientApp/styles.scss"],
+            "styles": [
+              "node_modules/@a-ui/core/dist/main.min.css",
```

Comment:
Waarom zou je deze manier prefereren boven de CDN?

Reply:
Omdat bij package update van @a-ui/core dan de correcte css file wordt gebruikt zonder dat de link naar CDN aangepast moet worden

Reply:
Ok, ik snap dat dit dan gemakkelijker is voor de developer, maar de CDN garandeert net de nodige caching voor eindgebruikers, waardoor laadtijden aanzienlijk versnellen, dus ik zou toch opteren om de originele set-up te bewaren.

Reply:
Aanpassingen zijn teruggedraaid.